### ISSUE TITLE
What about dev-****?

### ISSUE THREAD

**Issue by j0k3r** [pyrech/composer-changelogs#3]
When switching from a tagged version to a branch version, the generated text is wrong:

```
- fin1te/safecurl updated from v1.1 to dev-master
   See changes: fin1te/safecurl@v1.1...dev-master
   Release notes: https://github.com/fin1te/safecurl/releases/tag/dev-master
```

I think if the tag is starting with `dev-` you should :
- [ ] remove `dev-` from the _See changes_ section
- [ ] target `REPO/tree/BRANCHNAME` from the _Release notes_ section. Or maybe just remove this section.

**Comment by pyrech** [comment]
Thank you for your issue.

I'm going to refactore the plugin soon to support new installed package and
removed one (#2). So I will fix your issue at the same time!

Thanks again for your interest.
Le 30 sept. 2015 21:04, "Jeremy Benoist" <notifications@github.com> a
écrit :

**Comment by pyrech** [comment]
Hi :)

Finally got time to fix this issue properly. In the end, I used the short commit hash for compare urls instead of the branch name.

This is now released in v1.1.0. Thank you ;-)

**Comment by j0k3r** [comment]
:+1:
